### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by od-hunter**

### CODE REVIEW

**Comment by od-hunter** [comment]
Resolves #4

**Comment by od-hunter** [comment]
@Khauthymo , @Goodness5 please review

**Review by Goodness5 — CHANGES_REQUESTED** [review]
the flow of agreement and the screen designs do not fit in with the figma designs can you please adjust that
here is figma link in case you missed it https://www.figma.com/design/3nZ7dFPCVz4fdnfP5ZGlL6/Custos-Diretriz?node-id=1659-3562&node-type=section&t=4hOawtXyXTHtfd44-0

also the icon on no-agreement screen isan animated icon you should see the link to the gif version on the figma to lottiefiles.com

**Comment by faytey** [comment]
@od-hunter  are you still working on this issue

**Review by faytey — COMMENTED** [review]
- [ ] Kindly remove the contribution guidelines copy from the folder
- [ ] Also remove the .gitignore copy file present in your folder
- [ ] Kindly Implement all screens for the mobile app.
- [ ] Also the implementation/interaction and reading from the smart contract has not been worked on, kindly work on that.

File: app/(screens)/index.tsx
Comment (faytey) [inline app/(screens)/index.tsx:1]: Kindly remove commented blocks of code from the file

**Comment by od-hunter** [comment]
> @od-hunter are you still working on this issue

Yes I am ma. Will make necessary corrections an dm push. Thanks.

**Review by faytey — CHANGES_REQUESTED** [review]
- [ ] Kindly remove the contribution guidelines copy from the folder
- [ ] Also remove the .gitignore copy file present in your folder
- [ ] Kindly Implement all screens for the mobile app.
- [ ] Also the implementation/interaction and reading from the smart contract has not been worked on, kindly work on that.

**Review by faytey — CHANGES_REQUESTED** [review]
Kindly remove commented blocks of code from the file

**Comment by Goodness5** [comment]
@od-hunter kindly note that tomorrow is the last day for the od-hack no PRs submitted after tomorrow will be considered for the od-hack

**Comment by od-hunter** [comment]
> @od-hunter kindly note that tomorrow is the last day for the od-hack no PRs submitted after tomorrow will be considered for the od-hack

I’ll push a pr shortly ser, sorry for the delay please. I’m on it presently.

**Review by faytey — CHANGES_REQUESTED** [review]
- [ ] Awesome work, there's still seems to be some merge conflicts, please fix all merge conflicts between your branch and the base branch, then I can merge
- [ ] Also add screenshots or videos of the working app from your local machine thanks

**Review by faytey — CHANGES_REQUESTED** [review]
- [ ] Awesome work, there's still seems to be some merge conflicts, please fix all merge conflicts between your branch and the base branch, then I can merge

- [ ] Also add screenshots or videos of the working app from your local machine thanks

**Comment by od-hunter** [comment]
Uploading 4ee22e47-06bb-4fa3-9127-4c7c453c883b.mov…

@faytey , please review

**Comment by od-hunter** [comment]
https://github.com/user-attachments/assets/e57c629e-0d67-4dc9-ba35-5222cf7f1d66

@faytey , please review

**Review by faytey — APPROVED** [review]

**Pull request merged**